app/src/services/account-management.service.ts

```diff
@@ -376,7 +402,7 @@ export class UserLoginService {
       // first, load the valid tokens cached in the local store, if they are available
       // see: https://github.com/aws/amazon-cognito-identity-js/issues/71
       let cognitoUser = CognitoUtil.getCognitoUser();
-      cognitoUser.getSession( (err: Error, session: any) => {
+      cognitoUser.getSession((err: Error, session: any) => {
```

Comment:
We were not sure how to ensure credentials are refreshed properly upon expiration within 1 hour. Would you mind checking this to see if it is coded correctly to handle this?